### PR TITLE
Added Fibonacci number and Fibonacci series

### LINKED ISSUE #1259: Fibonacci numbers in C, C++, Java, Python and go
1) Finding nth Fibonacci numbers
2) printing nth Fibonacci numbers series

### PR DESCRIPTION
**Pull request by somya-kapoor**
Fixes #1259

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:
- Added Fibonacci numbers and Fibonacci series in C, C++, Java, Python and go

#### Languages Used:
- C
- C++
- python
- java
- go

#### Files Added:
- Fibonacci_Number.c
- Fibonacci_Number.cpp
- Fibonacci_Number.java
- Fibonacci_Number.py
- Fibonacci_Number.go

### CODE REVIEW

File: Fibonacci_Number/Fibonacci_Number.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
```
Comment (marwahmanbir) [inline Fibonacci_Number/Fibonacci_Number.c]: Remove space after ``include``.

File: Fibonacci_Number/Fibonacci_Number.cpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
+#include <iostream>
```
Comment (marwahmanbir) [inline Fibonacci_Number/Fibonacci_Number.cpp]: Remove space after ``include``.

File: Fibonacci_Number/Fibonacci_Number.cpp [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+int FibonacciSeries (int num)  
+{  
+    int a = 0, b = 1, i;  
+
+    if (num < 1)  
+        return 0;  
+
+    for (i = 1; i <= num; i++)  
+    {  
+        cout << b << " ";
```
Comment (marwahmanbir) [inline Fibonacci_Number/Fibonacci_Number.cpp]: Remove space around ``<<`` & ``>>`` in cout & cin statements.

Comment (somya-kapoor) [reply Fibonacci_Number/Fibonacci_Number.cpp]: But in other pr, mentors told me to add space around <<.

Comment (marwahmanbir) [reply Fibonacci_Number/Fibonacci_Number.cpp]: https://www.codesdope.com/cpp-gear-up/

Comment (marwahmanbir) [reply Fibonacci_Number/Fibonacci_Number.cpp]: Also nowhere mentioned in coding conventions (https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html)

**Comment by somya-kapoor** [comment]
@marwahmanbir  Done please review.

File: Fibonacci_Number/Fibonacci_Number.cpp (hunk 3 above)
Comment (somya-kapoor) [reply Fibonacci_Number/Fibonacci_Number.cpp]: Done please review.

File: Fibonacci_Number/Fibonacci_Number.cpp [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        a = b;  
+        b = c;  
+    }  
+} 
+
+int main() 
+{ 
+    int num; 
+    cin>>num;
+    cout<<num<<"th fibonacci number is "<<FibonacciNumber(num);
```
Comment (MastersAbh) [inline Fibonacci_Number/Fibonacci_Number.cpp]: Please find some better print statement,
as *th* won't go with all the numbers.

for 1, it should be 1st and not 1th

File: Fibonacci_Number/Fibonacci_Number.java [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import java.util.Scanner;
+
+class fibonacci
```
Comment (MastersAbh) [inline Fibonacci_Number/Fibonacci_Number.java]: You might want to make class name Fibonacci, with F uppercase

**Comment by somya-kapoor** [comment]
@MastersAbh Done please review

**Comment by somya-kapoor** [comment]
@marwahmanbir Done, Please review.

**Comment by somya-kapoor** [comment]
@MastersAbh Done please review

**Pull request merged**